src/KafkaFlow.Abstractions/Configuration/IClusterConfigurationBuilder.cs

```diff
@@ -25,6 +25,13 @@ public interface IClusterConfigurationBuilder
         /// <param name="handler">A handler to sets the values</param>
         /// <returns></returns>
         IClusterConfigurationBuilder WithSecurityInformation(Action<SecurityInformation> handler);
+        
+        /// <summary>
+        /// Configures schema registry
+        /// </summary>
+        /// <param name="configuration">Schema registry configuration</param>
+        /// <returns></returns>
+        IClusterConfigurationBuilder WithSchemaRegistry(Action<SchemaRegistryConfiguration> configuration);
```

Comment:
If you move this method to KafkaFlow package as an extension method, the implementation will be easier.